socorro/unittest/processor/test_processor_app.py

```diff
@@ -277,6 +277,48 @@ def mocked_save_raw_and_processed(*_):
         assert captured_exception.__class__ == ValueError
         assert captured_exception.args[0] == 'Someone is wrong on the Internet'
 
+    @mock.patch('socorro.lib.raven_client.raven')
+    def test_transform_get_error_with_raven_configured_successful(self, mock_raven):
+        # First mock everything so we're not sending real things anywhere
+        captured_exceptions = []  # a global
+
+        def mock_capture_exception(exc_info=None):
+            captured_exceptions.append(exc_info)
+            return 'someidentifier'
+
+        raven_mock_client = mock.MagicMock()
+        raven_mock_client.captureException.side_effect = mock_capture_exception
```

Comment:
Instead of using the global list, could we just assert that it was called with the proper argument?

```python
mock_raven_client = mock_raven.Client.return_value
mock_raven_client.captureException.assert_called_once_with(expected)
```

That'd save a lot of the boilerplate in this test.